bridges/snowbridge/primitives/inbound-queue/src/v2/message.rs

```diff
@@ -151,15 +174,17 @@ impl TryFrom<&Log> for Message {
 					});
 				},
 				1 => {
-					let foreign_data = IGatewayV2::AsForeignTokenERC20::abi_decode(&asset.data, true)
-						.map_err(|decode_err| {
-							log::debug!(
-								target: LOG_TARGET,
-								"decode foreign asset error {:?}",
-								decode_err
-							);
-							MessageDecodeError
-						})?;
+					let foreign_data =
+						IGatewayV2::AsForeignTokenERC20::abi_decode(&asset.data, true).map_err(
+							|decode_err| {
+								log::debug!(
```

Comment:
are these debug logs for decoding data actually useful? is returning the `MessageDecodeError` not enough?

Reply:
That's a good point, they were helpful during development but not useful anymore. Addressed in https://github.com/vgeddes/polkadot-sdk/pull/13/commits/242d9ee4310ac6c494a9329e66c66de4460197ee.